packages/react-server-webpack-plugin/src/test/react-server-webpack-plugin.test.ts

```diff
@@ -1,65 +1,71 @@
 import path from 'path';
 import webpack, {Compiler} from 'webpack';
-
-const generatedFileComment =
-  '// Generated by @shopify/react-server-webpack-plugin';
+import {HEADER} from '../react-server-webpack-plugin';
 
 const BUILD_TIMEOUT = 10000;
 
 describe('react-server-webpack-plugin', () => {
-  it(
-    'generates the server and client entrypoints when they do not exist',
-    async () => {
-      const [serverResults, clientResults] = await runBuild('no-entrypoints');
-
-      const serverModule = serverResults.modules.find(
-        ({name}) => name === './server.js',
-      );
-      const clientModule = clientResults.modules.find(
-        ({name}) => name === './client.js',
-      );
-      expect(serverModule.source).toMatch(generatedFileComment);
-      expect(clientModule.source).toMatch(generatedFileComment);
-    },
-    BUILD_TIMEOUT,
-  );
-
-  it(
-    'uses process.env to default port and host',
-    async () => {
-      const [serverResults] = await runBuild('no-entrypoints');
-
-      const serverModule = serverResults.modules.find(
-        ({name}) => name === './server.js',
-      );
-
-      expect(serverModule.source).toMatch('ip: process.env.REACT_SERVER_IP');
-      expect(serverModule.source).toMatch(
-        'port: process.env.REACT_SERVER_PORT',
-      );
-    },
-    BUILD_TIMEOUT,
-  );
-
-  it('does not use the generated client module when a bespoke file is present', async () => {
-    const [serverResults, clientResults] = await runBuild('client-entrypoint');
+  describe('node', () => {
+    it(
+      'generates the server and client entrypoints when they do not exist',
+      async () => {
+        const [serverResults, clientResults] = await runBuild(
+          'no-entrypoints',
+          true,
+        );
+
+        const clientModule = clientResults.modules.find(({name}) =>
+          name.includes('./client.js'),
+        );
+
+        const serverModule = serverResults.modules.find(({name}) =>
+          name.includes('./server.js'),
+        );
+
+        expect(clientModule.source).toMatch(HEADER);
+        expect(serverModule.source).toMatch(HEADER);
```

Comment:
Do we actually care about the header?

Reply:
It's how we verify that the plugin ran. What alternatives did you have in mind?

Reply:
Wouldn't any contents prove that? In the end, a user doesn't care about any of these things, so I'd rather see tests that actually show you how a user would experience the plugin, but at least let's make it more clear why this assertion means things are working.